src/headers.js

```diff
@@ -3,19 +3,19 @@ export class Headers {
     this.headers = headers;
   }
 
-  add(key, value){
```

Comment:
Bryan, would it be safe to assume that value is going to be of type string as well?